classes/class-wc-connect-nux.php

```diff
@@ -303,6 +342,26 @@ public function show_banner_after_connection() {
 			) );
 		}
 
+		public function show_tos_banner() {
+			if ( isset( $_GET['wcs-nux-tos'] ) && 'accept' === $_GET['wcs-nux-tos'] ) {
+				WC_Connect_Options::update_option( 'tos_accepted', true );
+				wp_safe_redirect( remove_query_arg( 'wcs-nux-tos' ) );
+				exit;
+			}
+			$this->show_nux_banner( array(
+				'title'          => __( 'Setup complete! We need you to accept our TOS', 'woocommerce-services' ),
+				'description'    => __( 'Everything is ready to roll, we just need you to agree to our Terms of Service.', 'woocommerce-services' ),
+				'button_text'    => __( 'I accept the TOS!', 'woocommerce-services' ),
+				'button_link'    => add_query_arg( array(
+					'wcs-nux-tos' => 'accept',
+				) ),
+				'image_url'      => plugins_url(
+					'images/nux-printer-laptop-illustration.png', dirname( __FILE__ )
+				),
+				'should_show_jp' => false,
```

Comment:
Doesn't have to happen in this PR I suppose, but when we finalize the copy for this notice we should consider an additional parameter like `should_show_terms` so we can show the TOS/privacy links without the JP logo.